lm_eval/tasks/aquarat.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+from tqdm import auto as tqdm_lib
+from . common import HFTask, simple_accuracy_metric, yesno
+from lm_eval.base import rf, mean
+
+class AquaRat(HFTask):
+    
+    def __init__(self):
+        self.download()
+        
+    def download(self):
+        if os.path.exists('data/aquarat/'):
+            pass
+        else:
+            sh( """
+                mkdir -p data/aquarat
+                wget https://raw.githubusercontent.com/deepmind/AQuA/master/train.json -O data/aquarat/aquarat-train.json
+                wget https://raw.githubusercontent.com/deepmind/AQuA/master/dev.json -O data/aquarat/aquarat-valid.json
+                wget https://raw.githubusercontent.com/deepmind/AQuA/master/test.json -O data/aquarat/aquarat-test.json
+                """)
+
+    def has_training_docs(self):
+        return True
+
+    def has_validation_docs(self):
+        return True
+
+    def has_test_docs(self):
+        return True
+
+    def fewshot_description(self):
+        # TODO: Incorporate rationale and compute using BLEU score
+        return "Read the following multiple choice questions and pick the correct choice"
+
+    def doc_to_text(self, doc):
+        options = " ".join(doc['options'])
+        return f"{doc['question']}\n{options}\nanswer: "
+    
+    def doc_to_target(self, doc):
+        return f"{doc['correct']}"
```

Comment:
For multiple choice problems, we should try to always predict the correct *answer* rather than the letter corresponding to the answer. See the SAT task (https://github.com/EleutherAI/lm-evaluation-harness/blob/master/lm_eval/tasks/sat.py) for an example of how to implement that.

Reply:
I'm not sure about this, because the original task is more about predicting the correct choice and then giving an explanation about that?